### ISSUE TITLE
Send user to Login page after session timeout

### ISSUE THREAD

**Issue by benhaynes** [directus/directus#190]
When idle for too long, you get logged out... you should be sent directly to the login page. Currently you get an error unless you refresh the page.

**Comment by Radiergummi** [comment]
It feels wrong to dig this issue out, but it describes what I've just experienced after upgrading from 6.3.9 to 6.4.2: After leaving the page open a while, clicking on any navigation element results in a toast saying `Unknown error` until I refreshed the page.  
I'll try to gather logs once this happens again.

**Comment by benhaynes** [comment]
Haha, no problem! I (sometimes) get the same thing... so let's open this back up! We need to make sure that if the session/cookie is logged out then there are NO errors... it simply should redirect you to the login page.

**Comment by WellingGuzman** [comment]
This issue may has been due to cache. @Radiergummi did you clear the cache after you upgrade?

**Comment by WellingGuzman** [comment]
Is there any errors in `api/logs`?

**Comment by Radiergummi** [comment]
@WellingGuzman I did indeed clear the cache since I experienced another issue after install - I still had an open session before upgrading. Once I tried to access the site after the installation, it was stuck saying "loading data", with an error showing up in the console - I think it was some session object being undefined.  
Clearing my session helped here... Maybe all existing sessions should be invalidated after upgrades?
Anyway so yes, I cleared the cache, and there are no errors in the log file from yesterday.
